.github/workflows/CI.yml

```
@@ -23,7 +23,7 @@ jobs:
           - { name: "cairo", features: "png,pdf,svg,ps,use_glib,v1_18,freetype,script,xcb,xlib,win32-surface", nightly: "--features 'png,pdf,svg,ps,use_glib,v1_18,freetype,script,xcb,xlib,win32-surface'", test_sys: true }
           - { name: "gdk-pixbuf", features: "v2_42", nightly: "--all-features", test_sys: true }
           - { name: "gio", features: "v2_74", nightly: "--all-features", test_sys: true }
-          - { name: "glib", features: "v2_74", nightly: "--all-features", test_sys: true }
+          - { name: "glib", features: "v2_74,serde", nightly: "--all-features", test_sys: true }
```

```suggestion
          - { name: "glib", features: "v2_74", nightly: "--all-features", test_sys: true }
          - { name: "glib serde", features: "v2_74,serde", nightly: "--all-features", test_sys: true }
```

or so maybe so we test both configurations?